### DISCUSSION TITLE
GLS(GBAS) Reciever

### DISCUSSION THREAD

**Discussion by CaptainGodual** [magknight/787aviatorsTracker#584] [Ideas]
Is it possible to get the GLS(GBAS) added to the magknight 787-9?

In real life this is a standard feature that is on all 787-9 Aircraft and some airport approaches require us to use GLS while rare it would be nice to have it in case it is needed.

**Comment by aeroniemi** [comment]
can probably add this with v12 - it'll have native GLS support along with sbas
